src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -231,33 +226,65 @@ private void OnBrowserConsoleMessage(object sender, ConsoleMessageEventArgs e)
             System.Diagnostics.Trace.Write(e.Message);
         }
 
-        internal static IDisposable SetupSearchModelEventsObserver(NodeSearchModel model, IEventController controller, int throttleTime = 200)
+        internal static IDisposable SetupSearchModelEventsObserver(NodeSearchModel model, IEventController controller, ILibraryViewCustomization customization, int throttleTime = 200)
         {
-            var observer = new EventObserver<NodeSearchElement, string>(
-                    nodes => controller.RaiseEvent("libraryDataUpdated", nodes),
-                    (s, e) => {
-                            var name = NodeItemDataProvider.GetFullyQualifiedName(e);
-                            return string.IsNullOrEmpty(s) ? name : string.Format("{0}, {1}", s, name);
-                        }
+            customization.SpecificationUpdated += (o,e) => controller.RaiseEvent("libraryDataUpdated");
```

Comment:
I see, but if OnSpecificationUpdate is handled after the event handler here, then UI won't be updated.